x/pairing/keeper/staking.go

```diff
@@ -182,6 +182,13 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, validator, creator, chainID strin
 		DelegateCommission: delegationCommission,
 	}
 
```

Comment:
missed to add the same on the flow for an existing entry

Reply:
done